src/components/MaButton/MaButton.vue

```diff
@@ -72,6 +72,15 @@ export default Vue.extend({
      * Sets the component displaying style
      * @values primary, secondary, white, gradient, no-background, no-background-white
      */
+
+    /**
+     * Sets the button font-size style
+     */
+    size: {
+      type: String,
+      default: 'small',
+      validator: (val) => ['small', 'large'].includes(val),
+    },
```

Comment:
```suggestion
    size: {
      type: String,
      default: 'small',
      validator: (val) => ['small', 'large'].includes(val),
    } as PropOptions<ButtonSize>,
```

In order to keep this TS inference, maybe we need to create this type on top of compoment. What do you think about it? :)

Reply:
Yes! I just forgot it... it didn't give me an error... weird...

Reply:
Maybe not an error. But if we want the IntelliSense features for autocomplete. I think we need it